packages/react-log-viewer/src/LogViewer/LogViewerToolbar.tsx

```diff
@@ -0,0 +1,108 @@
+import React, { useState, useEffect } from 'react';
+import { NUMBER_INDEX_DELTA } from './utils/constants';
+import { SearchInput, Toolbar, ToolbarItem, ToolbarContent, ToolbarGroup } from '@patternfly/react-core';
+import styles from '@patternfly/react-styles/css/components/LogViewer/loggerToolbar.styles';
+
+export interface LoggerToolbarProps extends React.HTMLProps<HTMLDivElement> {
+  hasToolbar?: boolean;
```

Comment:
We will need to add comments to describe what each of these props does. In all component files.